content/nldi.md

```diff
@@ -2,7 +2,6 @@
 author: David Blodgett
 date: 2017-04-22
 slug: nldi-intro
-draft: True
```

Comment:
Actually, no that's fine...we can still make changes before actually clicking the push to prod button.